packages/forms-web-app/tests/unit/validators/appellant-submission/upload-application-schema.test.js

```diff
@@ -101,17 +101,17 @@ describe('validators/appellant-submission/upload-application-schema', () => {
       );
     });
 
-    it('should call the validateFileSize validator', () => {
-      fn('some value', {
+    it('should call the validateFileSize validator', async () => {
+      await fn('some value', {
         req: {
-          session: appealWithFile,
-          files: { a: { mimetype: MIME_TYPE_JPEG, size: 12345 } },
+          session: appealWithoutFile,
```

Comment:
did this change from `appealWithFile` to prove both cases across different tests?

Reply:
Yes, for more coverage.  Do you want it changed back?

Reply:
No, not changed back, but you could add the extra test whilst keeping the existing one:

```
    [appealWithFile, appealWithoutFile].forEach((session) => {
      it('should call the validateFileSize validator', async () => {
        await fn('some value', {
          req: {
            session,
            files: { a: { mimetype: MIME_TYPE_DOC, size: 12345 } },
          },
          path: 'a',
        });
        expect(validateFileSize).toHaveBeenCalledWith(
          12345,
          config.fileUpload.pins.uploadApplicationMaxFileSize
        );
      });
    });
```